nncf/experimental/openvino_native/graph/model_transformer.py

```diff
@@ -52,22 +53,24 @@ def __init__(self, model: ov.Model):
         :param model: OpenVINO model to be transformed.
         """
         super().__init__(model)
-        self._model = model.clone()
-        self.name_to_node_mapping = {op.get_friendly_name(): op for op in self._model.get_ops()}
-        self._model_precision = ModelPrecision.FP32
-        for op in self._model.get_ops():
-            if op.get_type_name() == 'Constant':
-                if op.get_element_type().is_real():
-                    if op.get_element_type() == ov.Type(np.float16):
-                        self._model_precision = ModelPrecision.FP16
-                        break
+        self._model = model
 
+    @staticmethod
+    def _get_name_to_node_mapping(model: ov.Model) -> Dict[str, ov.Node]:
```

Comment:
This functionality repeats several times. If we need to update it we can update it only in one place.

Reply:
I think it will not be changed. My suggestion is to remove this method and use
```python
name_to_node_mapping = {op.get_friendly_name(): op for op in model.get_ops()}
```
directly.

Reply:
@KodiaqQ do you agree with rollback this change?

Reply:
@andrey-churkin is it really more convenient to write down this mapping for each new transformation instead of calling the method?

Reply:
I am okay with the current approach, may we keep it as it is?

Reply:
My position is following

I consider that the creation of such methods makes the code difficult to understand because when you read code you need to move inside this method and then move back.
If we use such an approach, we will get a lot of 1-2 line methods in the future.

I suggest @alexsu52 making the final decision in this conversation.

Reply:
Ok, ok, let's keep it in as was to avoid further unnecessary discussion because of the one line of code.

Reply:
I wanna keep it as it is. So, waiting the main judger - @alexsu52